ghost/minifier/lib/minifier.js

```diff
@@ -69,7 +65,7 @@ class Minifier {
     }
 
     async getMatchingFiles(src) {
-        return await glob(this.getFullSrc(src));
+        return glob.sync(this.getFullSrc(src));
```

Comment:
I know you said you didn't see a perf impact, but I am still _slightly_ nervous about this change. As far as I know the change to add this feature reduced boot performance significantly.

Tiny-glob claims to be a lot faster than node-glob but then it also claims to be have windows support (which is interesting).

Anyway all things considered I'd suggest we at least keep this async so that it doesn't block the boot process:

https://github.com/isaacs/node-glob/issues/420#issuecomment-698251894